cli/src/build.ts

```diff
@@ -494,6 +468,39 @@ export class BuildCommand extends Command {
       debug(`Dylib name: ${chalk.greenBright(cargoArtifactName)}`)
     }
 
+    const intermediateTypeFile = join(
+      tmpdir(),
+      `${cargoArtifactName}.napi_type_def.tmp`,
```

Comment:
`${cargoArtifactName}-${sha256(process.cwd()).substring(0, 7)}.napi_type_def.tmp` to prevent naming conflict?

Reply:
cool